### ISSUE TITLE
  Firmware 1.2-beta-0 Issues with downloading vids to device

### ISSUE THREAD

**Issue by BilliTheKid** [dride/dride-app-issues#35]
Keeps failing spratically. Maybe a server side bug?
![Screenshot_20191129-111237](https://user-images.githubusercontent.com/56843472/69866455-023d6e80-129c-11ea-923a-c3640fa04c0b.png)

**Comment by saoron** [comment]
Could you detail your actions step by step?

**Comment by BilliTheKid** [comment]
Clicked 3 dots, pop up comes up, press save to device error keeps occuring. Resetted device, was able to download. Tried again, stuck at 0% and wouldn't move then comes up with the error message. @saoron

**Comment by BilliTheKid** [comment]
@saoron Issue still there again today with downloading videos to my device and uploading to the cloud. To download a 1minute video is painfully slow. It timed out twice until I was able to finally download a 1 minute clip. Uploading to the cloud again painfully slow and could not achieve it as it stuck in 39% and again in 14% before the error page showed up. I'm on LTE all the time so it's definitely not my network.

**Comment by BilliTheKid** [comment]
Downloading video and uploading videos to cloud is painfully slow. Is this a firmware issue?

**Comment by BilliTheKid** [comment]
Today ran into issues upoading to cloud. First try it stopped at 33% and then gave an error. Tried it again and it stayed at 0% and kicked back an error then I finally got to upload a clip but it takes so so long but the dates are completely wrong. Fine when on the device but when uploaded to the cloud it changes date and time. Very strange @saoron 
![Screenshot_20191204-215812](https://user-images.githubusercontent.com/56843472/70187023-2efee500-16e5-11ea-933f-f73d4d873762.png)
![Screenshot_20191204-222145](https://user-images.githubusercontent.com/56843472/70187035-3625f300-16e5-11ea-97a3-baa447fed51a.png)
